### DISCUSSION TITLE
💯 Valeur métier

### DISCUSSION THREAD

**Discussion by seba1204** [photonsquid/7map#42] [Announcements]
Afin d'être en mesure de calculer la vélocité de notre projet à l'instant `t`, nous devons ajouter une valeur métier à chaque feature que nous implémentons.  Je n'ai pas trouvé de fonction built-in dans Github pour faire ça, du coup j'ai renommé toutes les issues de la team data en rajoutant à la fin entre parenthèses la valeur métier du la feature (précédée de l'emoji 💯).
J'ai aussi mis à jour les issue template, de manière à ce qu'on n'ait pas à copier-coller cet emoji à chaque nouvelle issue.
Si vous avez des commentaires, n'hésitez pas.
@7map/rendering  vous pouvez faire pareil, ça serait cool.
TODO: créer un workflow à chaque nouvelle issue closed qui calcule automatiquement la somme des valeurs métier réalisées (vélocité) et l'ajoute dans le README.md général.

**Comment by BSoDium** [comment]
Bonne idée, en revanche je trouve qu'un symbole un peu plus "formel" serait préférable (comme 💲 à  titre d'exemple). Je vais m'occuper de l'actualisation du format des issues côté rendering. 
En ce qui concerne le workflow, je pense qu'il faut prioriser le développement de l'application pour le moment; on pourra se pencher dessus un peu plus tard non ?

**Reply by seba1204** [reply]
Nickel, on adopte donc le dollar (💲).
> En ce qui concerne le workflow [...] on pourra se pencher dessus un peu plus tard non ?

Yes bien sûr !
